src/views/AdditionalInformation/RelatedRequestsTable.test.tsx

```diff
@@ -116,9 +116,9 @@ describe('Related Requests table', () => {
     expect(await screen.findByRole('heading', { name: 'Related requests' }));
 
     // this should NOT be in rendered if there is incoming table data
-    expect(() =>
-      screen.getByText('There are no additional requests linked to this system')
```

Comment:
Using screen was just fine
https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen

Sorry for the back and forth nits. More interested in being informative than calling out

Reply:
no apology necessary at all - i really appreciate the attention to detail and want to get this right! thanks Adam